tests/integration-tests/tests/createami/test_createami/test_build_image_custom_components/image.config.yaml

```diff
@@ -0,0 +1,11 @@
+Build:
+  InstanceType: {{ instance_type }}
+  ParentImage: {{ parent_image }}
+  Components:
+    {% if region == "us-east-1" %}
```

Comment:
Could you add a comment here? Describing what we're testing in the two cases

Reply:
done